plugins/inputs/cassandra/cassandra.go

```diff
@@ -274,25 +273,26 @@ func (c *Cassandra) Gather(acc telegraf.Accumulator) error {
 				m = newCassandraMetric(serverTokens["host"], metric, acc)
 			} else {
 				// unsupported metric type
-				log.Printf("I! Unsupported Cassandra metric [%s], skipping",
-					metric)
+				acc.AddError(fmt.Errorf("E! Unsupported Cassandra metric [%s], skipping",
+					metric))
 				continue
 			}
 
 			// Prepare URL
 			requestUrl, err := url.Parse("http://" + serverTokens["host"] + ":" +
 				serverTokens["port"] + context + metric)
 			if err != nil {
-				return err
+				acc.AddError(err)
+				continue
 			}
 			if serverTokens["user"] != "" && serverTokens["passwd"] != "" {
 				requestUrl.User = url.UserPassword(serverTokens["user"],
 					serverTokens["passwd"])
 			}
 
 			out, err := c.getAttr(requestUrl)
-			if out["status"] != 200.0 {
-				fmt.Printf("URL returned with status %v\n", out["status"])
```

Comment:
This change I'm not completely certain about as I'm not familiar with Cassandra. The code I removed appears like it would never evaluate as non-200 responses are handled up on [line 211](https://github.com/influxdata/telegraf/blob/c8cc01ba6a27a68a5d161152d9323d3e801edfbd/plugins/inputs/cassandra/cassandra.go#L211).
However if cassandra will respond with an http-200, but a non-200 in the body, then that would trigger the code. However the tests do not mock out a `status` field in the body.